docs/Running-behind-apache.md

```diff
@@ -253,6 +253,47 @@ DOSPageCount 30
 Restart Apache with `sudo service apache2 restart`, or the appropriate method to restart services on your system, to
 reload its configuration with your new values.
 
+
+### Virtual host
+To adjust the `DOSPageCount` for a specific virtual host, open your virtual host config, which can be found at
+`/etc/httpd/conf/sites-available/my-domain.conf` or `/etc/apache2/sites-available/my-domain.conf` and add the
+following:
+
+```conf
+# Increase the DOSPageCount to prevent 403 errors and IP addresses being blocked.
+<IfModule mod_evasive20.c>
+	DOSPageCount        30
+</IfModule>
+```
+
+For example:
+
+```conf
+<VirtualHost *:80>
+	RewriteEngine On
+	ProxyRequests Off
+	ProxyPreserveHost On
+	
+	ServerName netdata.domain.tld
+
+	<Proxy *>
+		AllowOverride None
+		AuthType Basic
+		AuthName "Protected site"
+		AuthUserFile /etc/apache2/.htpasswd
+		Require valid-user
+	</Proxy>
+
+	ProxyPass "/" "http://localhost:19999/" connectiontimeout=5 timeout=30 keepalive=on
+	ProxyPassReverse "/" "http://localhost:19999/"
+
+	# Increase the DOSPageCount to prevent 403 errors and IP addresses being blocked.
+	<IfModule mod_evasive20.c>
+		DOSPageCount        30
+	</IfModule>
+</VirtualHost>
+```
```

Comment:
i suggest to remove all specific parametes and leave only

```console
	<IfModule mod_evasive20.c>
		DOSPageCount        30
	</IfModule>
```

This is about `mod_evasive` all these `RewriteEngine On` is not related and just a noise.

Reply:
That's a good point. So, maybe the example could just be this:

---

... and add the following:

```conf
<VirtualHost *:80>
        ...
	# Increase the DOSPageCount to prevent 403 errors and IP addresses being blocked.
	<IfModule mod_evasive20.c>
		DOSPageCount        30
	</IfModule>
</VirtualHost>
```

Reply:
something like that, yes

Reply:
I was already doubting whether to do that, but since configuration got repeated a lot, I just followed that ¯\_(ツ)_/¯